### PR TITLE
[Doctest] Add `configuration_bert.py`

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Add `configuration_bert.py` to `utils/documentation_tests.txt` for doctest.

This PR will be used as a template for a new sprint.

### CODE REVIEW

File: src/transformers/models/bert/configuration_bert.py [hunk 1]
```diff
@@ -123,12 +123,12 @@ class BertConfig(PretrainedConfig):
     Examples:
 
     ```python
-    >>> from transformers import BertModel, BertConfig
+    >>> from transformers import BertConfig, BertModel
```
Comment (ydshieh) [inline src/transformers/models/bert/configuration_bert.py:126]: Change the import order of the model and configuration classes

File: src/transformers/models/bert/configuration_bert.py [hunk 2]
```diff
@@ -123,12 +123,12 @@ class BertConfig(PretrainedConfig):
...
 
     >>> # Initializing a BERT bert-base-uncased style configuration
     >>> configuration = BertConfig()
 
-    >>> # Initializing a model from the bert-base-uncased style configuration
+    >>> # Initializing a model (with random weights) from the bert-base-uncased style configuration
```
Comment (ydshieh) [inline src/transformers/models/bert/configuration_bert.py:131]: Add `(with random weights)` in the  comment before model initialization

**Review by sgugger — APPROVED** [review]
Thanks for clarifying the example!

**Review by LysandreJik — APPROVED** [review]

**Review by NielsRogge — APPROVED** [review]
Looks better :) thanks!

**Pull request merged**